tests/test_translator.py

```diff
@@ -277,6 +281,232 @@ def test_parse_raw_request_no_path(self):
         with self.assertRaises(ValueError):
             translator.parse_raw_request(raw_request)
 
+    def test_process_arguments_with_no_arguments(self):
+        # Arguments
+        args = argparse.Namespace(
```

Comment:
Well, this can be avoided by making a function over it, and passing only the required attributes.
Everything else, would be set to ``None``. Then the updation part will be bypassed. 
We can write separate tests for parser(which is what principle of unit testing says)